git_runner.py

```diff
@@ -3,5 +3,10 @@
 
 class GitRunner:
 
-    def __init__(self):
-        pass
+    def __init__(self, timeline):
+        self._timeline = timeline
```

Comment:
i don't think there's a need to name variables starting with _. it should be reserved for naming private methods i think.